### DISCUSSION TITLE
Xbox stream starts but drop too many frames... ideas?

### DISCUSSION THREAD

**Discussion by LuigiDic** [unknownskl/xbox-xcloud-client#1213] [General]
Can help?

**Comment by shashanknimje** [comment]
Okay, so open greenlight, then 

1. Go to Settings < Streaming
2. Under Streaming set both xCloud and xHome streaming bitrate to 12mbps
3. Set H.264 Profile to High

As per this YouTube recommended bitrate guide, you should shoot for 12mbps for H.264 1080p @60fps. All my packet loss issues pretty much disappeared. And now I have no stutter while streaming.

https://support.google.com/youtube/answer/2853702?hl=en

Edit: 10mbps seems to work best.
